tests/ImageSharp.Tests/Formats/Jpg/JpegDecoderTests.Metadata.cs

```diff
@@ -42,15 +42,16 @@ public partial class JpegDecoderTests
         };
 
         public static readonly TheoryData<string, int, int, PixelResolutionUnit> RatioFiles =
-        new TheoryData<string, int, int, PixelResolutionUnit>
+        new()
         {
             { TestImages.Jpeg.Baseline.Ratio1x1, 1, 1, PixelResolutionUnit.AspectRatio },
             { TestImages.Jpeg.Baseline.Snake, 300, 300, PixelResolutionUnit.PixelsPerInch },
-            { TestImages.Jpeg.Baseline.GammaDalaiLamaGray, 72, 72, PixelResolutionUnit.PixelsPerInch }
+            { TestImages.Jpeg.Baseline.GammaDalaiLamaGray, 72, 72, PixelResolutionUnit.PixelsPerInch },
+            { TestImages.Jpeg.Issues.MultipleApp01932, 400, 400, PixelResolutionUnit.PixelsPerInch }
```

Comment:
This test image is rather big, its 9350x6617 pixels, we should rather use a smaller one.

Here is a small version with the same issue:
![Issue1932_small](https://user-images.githubusercontent.com/38701097/148772021-fae58817-19f2-411f-bde8-aabce6c44039.jpg)

Reply:
Ah excellent. Much better!